base_user_role/models/user.py

```diff
@@ -58,13 +63,11 @@ def set_groups_from_roles(self):
             if not user.role_line_ids:
                 continue
             group_ids = []
-            role_lines = user.role_line_ids.filtered(
-                lambda rec: rec.is_enabled)
+            role_lines = user._get_enabled_roles()
             for role_line in role_lines:
```

Comment:
You can just set this code on the same line for "role_line in user._get_enabled_roles()"